test/core/tsi/alts/frame_protector/alts_crypter_test.cc

```diff
@@ -354,28 +357,46 @@ static void create_random_alts_seal_crypter(
   size_t key_length = rekey ? kAes128GcmRekeyKeyLength : kAes128GcmKeyLength;
   uint8_t* key;
   gsec_test_random_array(&key, key_length);
-  gsec_aes_gcm_aead_crypter_create(key, key_length, kAesGcmNonceLength,
-                                   kAesGcmTagLength, rekey, server_crypter_seal,
-                                   nullptr);
-  gsec_aes_gcm_aead_crypter_create(key, key_length, kAesGcmNonceLength,
-                                   kAesGcmTagLength, rekey,
-                                   server_crypter_unseal, nullptr);
-  gsec_aes_gcm_aead_crypter_create(key, key_length, kAesGcmNonceLength,
-                                   kAesGcmTagLength, rekey, client_crypter_seal,
-                                   nullptr);
-  gsec_aes_gcm_aead_crypter_create(key, key_length, kAesGcmNonceLength,
-                                   kAesGcmTagLength, rekey,
-                                   client_crypter_unseal, nullptr);
+  ASSERT_EQ(
+      gsec_aes_gcm_aead_crypter_create(
+          std::make_unique<grpc_core::GsecKey>(
+              absl::MakeConstSpan(key, key_length), rekey),
+          kAesGcmNonceLength, kAesGcmTagLength, server_crypter_seal, nullptr),
```

Comment:
micro-nit: Can you add `/*error_details=*/` in front of `nullptr`?

Similarly below